README.md

```diff
@@ -1,2 +1,112 @@
+[ci-img]: https://img.shields.io/travis/ciena-frost/ember-test-utils.svg "Travis CI Build Status"
+[ci-url]: https://travis-ci.org/ciena-frost/ember-test-utils
+
+[cov-img]: https://img.shields.io/coveralls/ciena-frost/ember-test-utils.svg "Coveralls Code Coverage"
+[cov-url]: https://coveralls.io/github/ciena-frost/ember-test-utils
+
+[npm-img]: https://img.shields.io/npm/v/ember-test-utils.svg "NPM Version"
+[npm-url]: https://www.npmjs.com/package/ember-test-utils
+
+[![Travis][ci-img]][ci-url] [![Coveralls][cov-img]][cov-url] [![NPM][npm-img]][npm-url]
+
 # ember-test-utils
-A test-framework agnostic set of utilities to aid in testing components
+
+ * [Installation](#Installation)
+ * [Getting Started](#Getting Started)
+ * [Contributing](#Contributing)
+
+## Installation
+```
+ember install ember-test-utils
+```
+
+## Getting Started
+
+**ember-test-utils** provides a set of utilities to help you in testing Ember components.  Wether you're adding integration or unit tests, you can use `registryHelper` to stub out factories.  This library requires you use `ember-cli-mocha` and `ember-mocha` as your testing framework.
+
+### Using `registryHelper`
+
+`registryHelper` must be properly initialized with `beforeSetup` and destroyed with `teardown`
```

Comment:
Are these `ember-cli-mocha` concepts? I thought it would be `beforeEach` or `beforeAll` and `afterEach` or `afterAll` I'm unfamiliar with `beforeSetup` and `teardown`.